src/cascadia/TerminalApp/Pane.h

```diff
@@ -139,6 +150,14 @@ class Pane : public std::enable_shared_from_this<Pane>
     bool _Resize(const winrt::Microsoft::Terminal::Settings::Model::ResizeDirection& direction);
     bool _NavigateFocus(const winrt::Microsoft::Terminal::Settings::Model::FocusDirection& direction);
 
+    bool _IsAdjacent(const std::shared_ptr<Pane> first, const PanePoint firstOffset, const std::shared_ptr<Pane> second, const PanePoint secondOffset, const winrt::Microsoft::Terminal::Settings::Model::FocusDirection& direction) const;
+    FocusNeighborSearch _FindNeighborFromFocus(const winrt::Microsoft::Terminal::Settings::Model::FocusDirection& direction,

```

Comment:
Other nit: could we rename this to `_FindNeighborForPane`? The two methods added here (`_FindNeighborFromFocus` and `_FindFocusAndNeighbor`) are pretty similar name-wise. More distinct names would help the reader parse what the code's trying to do here.